ui.frontend/src/main/webpack/site/js/types/searchOptions.ts

```diff
@@ -1,5 +1,5 @@
-import { TabConfig } from '../components/searchTabs'
-import { SearchCallbacks } from './callbacks'
+import {TabConfig} from '../components/searchTabs'
```

Comment:
@igorsimovski we have apparently different formatting.

I am using prettier. May want to align this at some points with backend devs :p

Reply:
can you supply me with the formatter, so i can fix the formatting for FE files?